.github/workflows/script.sh

```diff
@@ -27,13 +27,6 @@ compile|clean)
     bazel build $BAZEL_OPTS //...
     ;;
 
-bin)
```

Comment:
I think it is useful to be able to run this script locally and get a lot of the same behaviour as the CI system.

Reply:
The relevant part of this snippet is `./docker-run.sh ${OS}-${OS_VERSION}`. That's what users can execute locally (or in a different CI). The motivation for removing it from here is that:

- Tarballs are not generated in `/tmp/releases` anymore, so L33-L34 are not required.
- `docker-run.sh` does not need to be called in-place, so L31 is not required.